src/test/java/org/opensearch/knn/index/KNNCircuitBreakerIT.java

```diff
@@ -33,7 +33,7 @@ private void tripCb() throws Exception {
         updateClusterSettings("knn.memory.circuit_breaker.limit", "1kb");
 
         // Create index with 1 primary and numNodes-1 replicas so that the data will be on every node in the cluster
-        int numNodes = Integer.parseInt(System.getProperty("cluster.number_of_nodes"));
+        int numNodes = Integer.parseInt(System.getProperty("cluster.number_of_nodes", "1"));
```

Comment:
Added default value for number of clusters as 1, this may be missing for remote tests and lead to NPE